components/ListComponents/ListComponents.module.scss

```
@@ -34,18 +92,43 @@
   display: flex;
+  flex-wrap: wrap;
   font-size: 1.625rem;
   font-weight: 600;
+  gap: 0.75rem;
   margin-bottom: 1rem;
 }
 
+.listNameText {
+  flex: 1 1 auto;
+  min-width: 0;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)